docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx

```diff
@@ -11,243 +11,438 @@ keywords:
-During plotting, there are both parallel and sequential parts of the table generation. By generating several tables simultaneously, we can overlap the sequential parts with parallel parts, thus improving CPU utilization. While generating tables for all records requires significant RAM, producing tables for only a few records at a time offers an optimal balance between CPU and RAM usage.
-We added the `--record-encoding-concurrency` option to override the default behavior, which allocates one record for every two cores but does not exceed eight in parallel. According to our internal testing with P-cores, E-cores, and combinations of P+E cores, this setting appears to achieve peak performance.
-If you prefer to use the previous behavior, or if your RAM usage becomes too high, you can set `--record-encoding-concurrency` to `1`. You may also experiment with setting it to `2`, `3`, etc., which may yield better results depending on your specific CPU/RAM configuration.
+:::tip[Understanding the DSN]
 
-### Create Missing Farms
+The DSN can be a complex topic. To gain a better understanding of our Decentralized Storage Network, please refer to this [guide](https://academy.autonomys.xyz/subspace-protocol/network-architecture/distributed-storage-network) from our Academy.
 
-A farmer has the option to specify whether the system should automatically create missing farms upon startup. If you provide a path to a plot that isn't found, the system will generate a new one. However, this may not be desirable if a drive fails to mount properly. 
-By default, this option is set to `true`, but you can override it by adding `--create false`. Setting this flag to `false` after establishing your plots can prevent unintentional file writes to the wrong drive.
+:::
+
+The default parameters are configured with common consumer modem and router capabilities in mind. Adjusting certain parameters can enhance DSN sync performance by increasing parallelism. However, if you choose to significantly increase these values, ensure that your modem or router can handle the additional traffic.
+
+```bash
+--dsn-out-connections <value>
+--dsn-pending-out-connections <value>
+```
 
-### Record Chunks Reading Mode
+</details>
 
-Upon startup, each farm will [benchmark](#benchmarking-your-farmer) the performance of every plot to identify the most efficient proving method, yielding either `ConcurrentChunks` or `WholeSector` results. If you already know the optimal method for your setup, you can specify it as an argument for each farm to save time benchmarking. 
-For example, you can include record-chunks-mode= after defining the path and size, assigning the desired value, e.g., `path=/mnt/subspace1,size=100G,record-chunks-mode=ConcurrentChunks`. If you do not provide this parameter, the system will benchmark each disk on startup to identify the most efficient method.
+<details>
+<summary><small>Snap Sync</small></summary>
```

Comment:
I think this description is for the full sync, and what this section should really be focusing on, is the snap sync. 
Randy has updated the description for the snap sync in this PR https://github.com/autonomys/subspace-docs/pull/692, do you mind adding it here? Feel free to leave everything that you've put about the full sync, it reads well